include/lsst/afw/image/PhotoCalib.h

```diff
@@ -0,0 +1,426 @@
+/*
+ * LSST Data Management System
+ * Copyright 2017 LSST Corporation.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+
+#ifndef LSST_AFW_IMAGE_PHOTOCALIB_H
+#define LSST_AFW_IMAGE_PHOTOCALIB_H
+
+/**
+ * The photometric calibration of an exposure.
+ *
+ * A PhotoCalib is a BoundedField (a function with a specified domain) that converts between calibrated
+ * counts-on-chip (ADU) to flux and magnitude. It is defined in terms of "maggies", which are a linear
+ * unit defined in SDSS: http://www.sdss.org/dr12/algorithms/magnitudes/#nmgy
```

Comment:
I've defined PhotoCalib in terms of Maggies. I've clarified the docstring here to note that the SDSS link gives the definition in terms of nanomaggies.